pkg/authentication/externalidp/awsiam/token_caching.go

```diff
@@ -25,15 +27,15 @@ type JWTCache struct {
 	FilePath string
 }
 
-func NewJWTCache(baseDir, lakectlDir, cacheDir, fileName string) (*JWTCache, error) {
+func NewJWTCache(baseDir, fileName string) (*JWTCache, error) {
 	if baseDir == "" {
 		var err error
 		baseDir, err = os.UserHomeDir()
 		if err != nil {
 			return nil, err
 		}
 	}
-	cachePath := filepath.Join(baseDir, lakectlDir, cacheDir)
+	cachePath := filepath.Join(baseDir, LakectlDirName, CacheDirName)
```

Comment:
Those sub diretories are irrelevant for this component. 
It doesn't need to **force** or even know about `LakectlDirName, CacheDirName`